### DISCUSSION TITLE
ADMIN USER AND OTHER USER

### DISCUSSION THREAD

**Discussion by mtran-88** [Ylianst/MeshCentral#6009] [Q&A] [answered]
Hi, when I set up MeshCentral, I created my own user of 'mtran'; added a group and devices to it using the agent.  I then created another user named 'admin' and tried to do the same by adding a group and devices to it.  The 'admin' user did not see any devices on it even though I've reloaded the agent on the remote device.  Is it by design that the first user needs to be added as 'admin'?  hope that makes sense.  Thx!

**Comment by si458** [comment]
meshcentral is designed per user!
so machines/groups you create on one user wont show on another,
to share then you need add permission for the group to that user using the admin account
OR
you set a config in config.json to share the groups (cant remember what it is, just looking now)

**Reply by mtran-88** [reply, marked as answer]
I found it!  It's under 'My Users' -> 'Add Device Group', then you can give that user certain access rights.
